judgments/forms/search_forms.py

```diff
@@ -21,20 +24,47 @@ def _get_choices_by_group(courts):
         "key2": "value2"
     }
     """
-    options: dict = {}
+    options: dict[Union[Optional[str], CourtParam], Union[str, dict[CourtParam, str]]] = {}
```

Comment:
Shall we move this `dict[Union[Optional[str], CourtParam], Union[str, dict[CourtParam, str]]]` to a type alias so we can use it here and on option_a/option_b?

Reply:
Agree with moving this to a concrete class, this looks like it's complex enough we want it to stand on its own.

Reply:
Done.